### PR TITLE
EZP-31733: U.I colors

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31733
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/scss/_custom.scss [hunk 1]
```diff
@@ -3,38 +3,51 @@ $ez-admin-transition: cubic-bezier(0.25, 0.8, 0.25, 1);
 $ez-admin-widget-open-transition: all 0.3s ease-in;
 $ez-admin-widget-close-transition: all 0.3s ease-out;
 
+/* Ibexa colors */
+$ibexa-color-red: #DB0032;
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_custom.scss:7]: Let's not introduce new variable for the moment.

File: src/bundle/Resources/public/scss/_custom.scss [hunk 2]
```diff
@@ -3,38 +3,51 @@ $ez-admin-transition: cubic-bezier(0.25, 0.8, 0.25, 1);
...
+
+$ez-secondary-ground-pale: #E0E0E8;
+$ez-ground-primary: #fff;
+$ez-secondary-ground: #DB0032;
+$ez-secondary-ground-medium: #DB0032;
+
+$ez-secondary-ground-dark: #DB0032;
+
 $ez-ground-primary-inverted-light: #445a64;
 $ez-ground-primary-inverted: #565e63;
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_custom.scss:52]: Please add colors for all variables

File: src/bundle/Resources/public/scss/_mixins.scss [hunk 3]
```diff
@@ -241,3 +239,11 @@
         line-height: calculateRem(36px);
     }
 }
+
+@mixin nav-item-active-hover {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_mixins.scss:243]: Please create a proper mixin file in the `mixins` folder and import it in this file.

File: src/bundle/Resources/public/scss/_custom.scss [hunk 4]
```diff
@@ -4,42 +4,42 @@ $ez-admin-widget-open-transition: all 0.3s ease-in;
...
-$ez-black: #333;
+$ez-color-base-dark: #131C26;
+$ez-black: #131C26;;
 
-$ez-secondary-ground-pale: #e1f5ff;
-$ez-ground-primary: #a8c8d5;
-$ez-secondary-ground: #2b84b1;
-$ez-secondary-ground-medium: #106d95;
-$ez-secondary-ground-dark: #0a5574;
-$ez-ground-primary-inverted-light: #445a64;
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_custom.scss:38]: Please do not remove variables. It's BC break.

File: src/bundle/Resources/public/scss/_main-nav.scss [hunk 5]
```diff
@@ -1,7 +1,7 @@
 .ez-main-nav {
     padding: 0;
-    background-color: $ez-ground-base-medium;
-
+    background-color: $ez-white;
+    border-bottom: calculateRem(1px) solid $ez-ground-base-dark;
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/scss/_main-nav.scss:4]: new line between props and new selector

File: src/bundle/Resources/public/scss/_tabs.scss [hunk 6]
```diff
@@ -44,11 +45,11 @@
     }
 
     .nav-tabs {
-        border-bottom: calculateRem(2px) solid $ez-ground-base-dark;
+        border-bottom: calculateRem(1px) solid $ez-ground-base-dark;
 
         &.ez-tabs {
             .nav-item {
-                margin-bottom: -2px;
+                margin-bottom: -1px;
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/scss/_tabs.scss:52]: calculateRem

File: src/bundle/Resources/public/scss/ui/modules/sub-items-list/_action.btn.scss [hunk 7]
```diff
@@ -21,23 +20,13 @@
...
-    }
-
-    &--trash {
-        border: calculateRem(1px) solid $ez-color-danger;
-        background: $ez-color-danger;
-
         &:not(.c-action-btn--disabled):hover,
         &:not(.c-action-btn--disabled):focus {
-            background: $ez-color-danger-hover;
+            svg {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/ui/modules/sub-items-list/_action.btn.scss:27]: The same

**Pull request merged**